### ISSUE TITLE
GUI issues

### ISSUE THREAD

**Issue by bovirus** [erickutcher/httpdownloader#235]
@erickutcher 

Please check the screenshots about GUI issues

![HTTP Downloader - GUI - 01](https://github.com/erickutcher/httpdownloader/assets/1262554/ec93d351-84f8-41ae-a838-5480dde47d5e)

![HTTP Downloader - GUI - 02](https://github.com/erickutcher/httpdownloader/assets/1262554/133d229a-191f-46be-8048-2b7e04c6fe92)

![HTTP Downloader - GUI - 03](https://github.com/erickutcher/httpdownloader/assets/1262554/0ebd494b-365e-4ccc-9951-f3d1d21502a5)

![HTTP Downloader - GUI - 04](https://github.com/erickutcher/httpdownloader/assets/1262554/3ab81423-5c91-44ae-91d9-0f4d4b4d9146)

**Comment by erickutcher** [comment]
I'll try to adjust the width and alignment of things. It's going to take a while though since everything is hard coded.

Also, make sure your translation uses the pluralized "URL(s)" where ever it's listed. For example: "Add URL(s)" -> "Aggiungi URL(s)"

**Comment by bovirus** [comment]
@erickutcher 

Version 1.0.5.2

Issue 01 - Not fixed
Issue 02 - Not fixed
Issue 03 - Not fixed
Issue 04 - Not fixed

Could you please fix them?

**Comment by bovirus** [comment]
@erickutcher 

In 1.0.5.3

Issue 1-3-4 fixed.
Issues 2 seems again in place.

**Comment by erickutcher** [comment]
I haven't merged your translations yet because it's missing a string on line 349: https://github.com/erickutcher/httpdownloader/pull/240

Download the latest code and resubmit the Pull request so it can be merged. Once I get the correct locale, I'll reupload the programs with it and it'll be correct.

**Comment by bovirus** [comment]
@erickutcher 

Now Italian language should be OK.
Please check and merge. Thanks.

**Comment by erickutcher** [comment]
I've merged your update and updated the downloads. You can redownload it again and it'll have the new locale file. https://github.com/erickutcher/httpdownloader/releases/download/v1.0.5.3/HTTP_Downloader_64.zip

**Comment by mp3sova** [comment]
Hello!
Win10, scaling 225%
https://i.imgur.com/NB21NXv.jpg

https://i.imgur.com/tjHnwnB.jpg

https://i.imgur.com/d8S2oFz.jpg

**Comment by erickutcher** [comment]
Yeah I added support for scaling in another program that I made and it was painful. If I add it to HTTP Downloader, then it's probably just going to use GDI scaling and it'll look aliased. It's too large of an undertaking at the moment.

**Comment by bovirus** [comment]
@erickutcher 

OK. Thanks for the try.

**Comment by erickutcher** [comment]
Here's a beta version you can try out:

[HTTP_Downloader_32.zip](https://github.com/erickutcher/httpdownloader/files/13836942/HTTP_Downloader_32.zip)
[HTTP_Downloader_64.zip](https://github.com/erickutcher/httpdownloader/files/13836943/HTTP_Downloader_64.zip)

I added gdiScaling to the program's manifest file and on Windows 10 and newer systems it'll scale most everything in proportion. It may look blurry at some percentages, but at least things aren't scrunched together.
